mail/api.py

```diff
@@ -154,16 +188,24 @@ def send_individual_email(cls, subject, body, recipient,  # pylint: disable=too-
             recipient (str): email recipient
             sender_address (str): Sender email address
             sender_name (str): Sender name
+            raise_for_status (bool): If true and a non-zero response was received,
 
         Returns:
             requests.Response: response from Mailgun
         """
         # Since .send_batch() returns a list, we need to return the first in the list
         responses = cls.send_batch(subject, body, [recipient], sender_address=sender_address, sender_name=sender_name)
-        return responses[0]
+        _, response, exception = responses[0]
+        if exception is not None:
+            raise exception
```

Comment:
I'm confused. So if an exception is raised in `send_batch()`, the exception is caught by the `except Exception` line. But then the caught exception is returned, so that it can be re-raised again here? This is going to make debugging these exceptions a nightmare. It would be better to just allow exceptions to raise, so that the stack trace doesn't get mucked up.